cookbooks/aws-parallelcluster-slurm/recipes/install.rb

```diff
@@ -17,6 +17,7 @@
 
 dns_domain "Install dns related packages"
 mysql_client 'Install mysql client'
+mysql_repo 'Configure mysql repository'
```

Comment:
should we configure the repo before installing mysql client?

Reply:
This was a design decision to keep control of the installed version and at the same time have the repository configured for taking future (security) updates if needed.